src/Utils/GraphsDetails.jsx

```diff
@@ -12,8 +12,8 @@ import { ReactComponent as AnatomicalFindings } from "../assets/icons/anatomical
 import { ReactComponent as WorldMap } from "../assets/icons/consciousness-world-map.svg";
 import { ReactComponent as Effects } from "../assets/icons/effects.svg";
 
-export const graphsHeaders = [
-  {
+export const graphsHeaders = {
+  "Theory Support Overview": {
```

Comment:
Not a good practice to use string with spaces as object key (because its hart to access it, not typed etc)